docs/content_credentials.md

```diff
@@ -78,7 +70,7 @@ A future lower level API will wrap the Claim and Store structures providing a si
 
 ## Asset objects (not directly related to the above)
 
-An Asset object creates a persistent layer over the asset_io traits. Currently we parse entire asset every time we need to access information about it. We have separate passes for XMP, JUMBF, Offset/box generation & etc..
+An Asset object creates a semi-persistent layer over the asset_io traits. Currently we parse entire asset every time we need to access information about it. We have separate passes for XMP, JUMBF, Offset/box generation & etc..
```

Comment:
Why is it *semi* persistent? Because in spirit it is a kind of cache to avoid multiple re-parsings?